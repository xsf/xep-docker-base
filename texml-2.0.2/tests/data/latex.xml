<?xml version="1.0" encoding="windows-1251"?>
<TeXML>&#169;&#xa0;Здравствуй,&#xa0;&#x2026;!
<cmd name="x">
<parm>Здравствуй,&#xa0;&#x2026;!</parm>
<opt>Здравствуй,&#xa0;&#x2026;!</opt>
</cmd>
Check if unicode.map bug fixed: [&#xa0;][&#xad;]
</TeXML>
