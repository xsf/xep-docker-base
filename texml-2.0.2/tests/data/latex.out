\textcopyright{}~Здравствуй,~\ldots{}! \x{Здравствуй,~\ldots{}!}[Здравствуй,~\ldots{}!]
Check if unicode.map bug fixed: [~][\-]
